addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -169,6 +177,7 @@ var InputField = DebouncedField.extend({
     events: _.extend({}, DebouncedField.prototype.events, {
         'input': '_onInput',
         'change': '_onChange',
+        'blur' : '_onBlur',
```

Comment:
Can't we include this event only in SMS widget instead of a void one in base ?

Reply:
Yes actually we could. I've made that choice for two reasons :
- Easier / less code to write down
- This way if it is needed in the futur it's already implemented in a more generic way (rather then repeating this process in other modules or so)

Reply:
Seems also legit. Maybe @aab-odoo has its idea on it ?